### PR TITLE
Add Sean's turbo shot

### PR DESCRIPTION
**Pull request by SeanDougherty**

### CODE REVIEW

**Comment by hbeberman** [comment]
☕

**Comment by SeanDougherty** [comment]
> ☕

Good point, I need to take this into consideration

File: turbo-shot.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+======
+This recipe is great for light roasted espresso
+
+## Ingredients
+* 15-17g of light roasted coffee
+
+## Tools
+* an espresso machine or pressurized/heated espresso extraction device (e.g., flair 58)
+* 18g basket
+* burr coffee grinder
```
Comment (hbeberman) [inline turbo-shot.md:11]: Flat or conical?

Comment (SeanDougherty) [reply turbo-shot.md:11]: yes

**Comment by SeanDougherty** [comment]
Updated to correct capitalization and improve detailing on burr grinder type

**Pull request merged**